website/docs/language/backend/azurerm.mdx

```diff
@@ -9,495 +9,501 @@ Stores the state as a Blob with the given Key within the Blob Container within [
     container_name       = "tfstate"                                  # Can be passed via `-backend-config=`"container_name=<container name>"` in the `init` command.
     key                  = "prod.terraform.tfstate"                   # Can be passed via `-backend-config=`"key=<blob key name>"` in the `init` command.
-    sas_token            = "abcdefghijklmnopqrstuvwxyz0123456789..."  # Can also be set via `ARM_SAS_TOKEN` environment variable.
   }
 }
 ```
 
-## Example Data Source Configurations
+## Access Key Lookup
 
-### Data Source: Azure AD User Principal via Azure CLI
+This method requires a valid Azure Active Directory principal and is a fallback for when Azure Active Directory authentication cannot be used on the storage account data plane.
 
-This method is not suitable for automation since it only supports a User Principal. To check which tenant and subscription you are pointed to, run `az account show`.
+This method queries the management plane to get the storage account Access Key and then uses that Access Key to authenticate to the storage account data plane. As such it requires elevated permissions on the storage account.
 
-*Connect to Storage Account with Access Key*
+This method is retained for backwards compatibility, but we do not recommend it for new workloads.
 
-```hcl
-data "terraform_remote_state" "foo" {
-  backend = "azurerm"
-  config = {
-    resource_group_name  = "StorageAccount-ResourceGroup"
-    storage_account_name = "terraform123abc"
-    container_name       = "tfstate"
-    key                  = "prod.terraform.tfstate"
-  }
-}
-```
+### Required Inputs
 
-*Connect to Storage Account with Azure Active Directory authentication*
+The following inputs are always required for this method:
 
-```hcl
-data "terraform_remote_state" "foo" {
-  backend = "azurerm"
-  config = {
-    resource_group_name  = "StorageAccount-ResourceGroup"
-    storage_account_name = "terraform123abc"
-    container_name       = "tfstate"
-    key                  = "prod.terraform.tfstate"
-    use_azuread_auth     = true                            # Can also be set via `ARM_USE_AZUREAD` environment variable.
-  }
-}
-```
+- `tenant_id` - The tenant ID of the Azure Active Directory principal is required to authenticate to the storage account management and data plane. If using Azure CLI, this can be inferred from the CLI session. This can also be set via the `ARM_TENANT_ID` environment variable.
+- `subscription_id` - The subscription ID of the storage account is required to query the management plane. If using Azure CLI, this can be inferred from the CLI session. This can also be set via the `ARM_SUBSCRIPTION_ID` environment variable.
+- `resource_group_name` - The resource group name of the storage account is required to query the management plane.
+- `storage_account_name` - The name of the storage account where the state file blob resides in.
```

Comment:
```suggestion
- `storage_account_name` - The name of the storage account to write the state file blob to.
```

Similar callout to the same wording above, if this is not accurate please feel free to discard this suggestion.